server/core.c

```diff
@@ -5316,15 +5316,24 @@ AP_DECLARE(int) ap_sys_privileges_handlers(int inc)
 
 static int check_errorlog_dir(apr_pool_t *p, server_rec *s)
 {
+    char *dir;
+    apr_finfo_t finfo;
+    apr_status_t rv;
+
     if (!s->error_fname || s->error_fname[0] == '|'
         || s->errorlog_provider != NULL) {
         return APR_SUCCESS;
     }
     else {
         char *abs = ap_server_root_relative(p, s->error_fname);
-        char *dir = ap_make_dirstr_parent(p, abs);
-        apr_finfo_t finfo;
-        apr_status_t rv = apr_stat(&finfo, dir, APR_FINFO_TYPE, p);
+        if (!abs) {
+            ap_log_error(APLOG_MARK, APLOG_STARTUP|APLOG_EMERG, 0,
+                          ap_server_conf, APLOGNO(02291)
+                         "Cannot construct error log file path '%s'", s->error_fname);
```

Comment:
can you include the input that triggers this? Hopefully in a way as simple as httpd.conf content rather than calling these functions directly.